tests/Unit/StoreApiTest.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Tests\Unit;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class StoreApiTest extends TestCase
+{
+	// Test it can store new user
+    public function test_it_can_store_new_user($name ="dummy name", $address="dummy address", $age="12")
```

Comment:
Why do you insert test data in arguments? This is the general way? I thought it's easier to insert inside the code.